### ISSUE TITLE
Compare Size of old Extracts with new Extracts

### ISSUE THREAD

**Issue by lukasmartinelli** [osm2vectortiles/osm2vectortiles#266]
This helps to get an approximation where the filesize is heading. Possible now that I have europe test data #251.

**Comment by lukasmartinelli** [comment]
Okay looks promising @klokan that we can get under 60GB.
A bit weird that the change is so different from extract to extract. But perhaps because tweaked so much at the mapping. 

| extract        | v1.0 (MB) | v1.5 (MB) | change (%) | 
|----------------|------|------|--------| 
| belgium        | 943  | 696  | -26    | 
| czech_republic | 1107 | 904  | -18    | 
| denmark        | 543  | 428  | -21    | 
| switzerland    | 649  | 504  | -22    | 
| zurich         | 34   | 31   | -10    | 
| paris          | 144  | 106  | -26    | 
| berlin         | 90   | 71   | -21    |

**Comment by manuelroth** [comment]
That looks really good, so a 10 - 20% decrease should be possible.

**Comment by klokan** [comment]
Awesome! Very glad to hear the good news!

Did you count the tiles on zoom 1-7 while comparing the extracts?

BTW Design of the mailer for the USB stick grows here:
https://www.pixelapse.com/klokantech/projects/OSM2VT%20USB/files

**Comment by klokan** [comment]
BTW how is it with the size of individual tiles (the 500K limit for upload to MapBox.com).
Is it better now?

**Comment by lukasmartinelli** [comment]
> Did you count the tiles on zoom 1-7 while comparing the extracts?

The lower zoom levels are merged into the v1.0 and v1.5 extracts.

> BTW how is it with the size of individual tiles (the 500K limit for upload to MapBox.com).
Is it better now?

Hmm not really. Every extract has like 5-7 tiles that are too big (500-700kB).
**Most of them are on z5 and z6**
This is kind of nasty for styling - this means people have to purge away the too large tiles for styling in Mapbox Studio.

```bash
# purge tiles larger than 500kb
mboptimize size <mbtiles_file> -s 500000
```
